gfx/common/x11_common.h

```diff
@@ -44,6 +44,10 @@ void x11_save_last_used_monitor(Window win);
 void x11_show_mouse(Display *dpy, Window win, bool state);
 void x11_windowed_fullscreen(Display *dpy, Window win);
 void x11_suspend_screensaver(Window win, bool enable);
+void x11_suspend_screensaver_xdg_screensaver(Window win, bool enable);
```

Comment:
One has to be the first.

They're not used outside that file, and shouldn't be. They're just implementation details. Not really important, just a cleanup.